src/DataDefinitionsBundle/Model/DataSetAwareTrait.php

```diff
@@ -20,14 +20,14 @@
 
 trait DataSetAwareTrait
 {
-    protected ImportDataSetInterface $dataSet;
+    protected ?ImportDataSetInterface $dataSet = null;
 
-    public function getDataSet(): ImportDataSetInterface
+    public function getDataSet(): ?ImportDataSetInterface
     {
-        return $this->dataSet;
+        return $this->dataSet ?? null;
```

Comment:
In some cases the getter is called before setting the `dataSet` property to any value. This change causes the getter to return null (like it did before) instead of throwing and exception. For this to function properly, the Interface, too, had to be modified.

Reply:
I don't like the interface change, but I guess since I rushed the typed properties, we should accept that for now and work on a fix with the next version